templates/_components/_image.twig

```diff
@@ -386,9 +386,10 @@
 
     {% if not (imageIsSvg or imageIsAnimatedGif) %}
       {% set imageTransformSizes = imagePreset.sizes ?? [{ width: _imageAsset.width }] %}
-      {% set imageTransformParams = {
-        position: _imageAsset.getFocalPoint(),
-      } | merge(imagePreset.transform ?? {}) %}
+      {% set imageTransformPosition = _imageAsset.getFocalPoint() is defined
```

Comment:
No need to create a new `imageTransformPosition` variable. This can also be called `imageTransformParams` itself.

Reply:
Okay. Done.